<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SecureInstantMessage</string>
  </property>
  <property name="windowIcon">
   <iconset resource="simImgResource.qrc">
    <normaloff>:/img/qq.ico</normaloff>:/img/qq.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>150</width>
       <height>18</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="remePswCheckBox">
        <property name="text">
         <string>记住密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="loginCheckBox">
        <property name="text">
         <string>自动登录</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="accountComboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="registerAccBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>84</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    
	border-color: rgb(141, 255, 70);
}</string>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pswLineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="changePswPushBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
	border-color: rgb(85, 255, 127);
}</string>
     </property>
     <property name="text">
      <string>更改密码</string>
     </property>
    </widget>
    <widget class="QToolButton" name="headImgLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="loginPushBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#loginPushBtn:hover
{
        border-image:url(:/img/LoginWidget/btnLoginHover.png) 0 0 0 0 stretch stretch;
}

QPushButton#loginPushBtn
{
	border:0px;
	padding:-1px;
	width:185px;
	height:49px;
        border-image:url(:/img/LoginWidget/btnLoginNormal.png) 0 0 0 0 stretch stretch;
}

QPushButton#loginPushBtn:pressed
{
        border-image:url(:/img/LoginWidget/btnLoginDown.png) 0 0 0 0 stretch stretch;
}</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="simImgResource.qrc"/>
 </resources>
 <connections/>
</ui>
